cypress/e2e/api/engageShere.cy.js

```diff
@@ -0,0 +1,144 @@
+describe('Listar clientes na aplicação EngageSphere', () => {
+  const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}`
+  it('Recuperar clientes com sucesso', () => {
+    cy.request('GET', `${CUSTOMERS_API_URL}customers?page=1&limit=10&size=All`).then(
+      (response) => {
+        expect(response.status).to.eq(200)
```

Comment:
E então, aqui você poderia fazer assim:

```suggestion
        expect(status).to.eq(200)
```